src/particles.cpp

```diff
@@ -455,6 +456,26 @@ void ParticleManager::stepSpawners (float dtime)
 	}
 }
 
+void ParticleManager::stepSpawnersLocal (float dtime)
+{
+	MutexAutoLock lock(m_spawner_local_list_lock);
```

Comment:
I just duplicated the way that the list of particlespawners sent by the server is handled.